Test/VSComp2010/Problem2-Invert.dfy

```diff
@@ -1,4 +1,4 @@
-// RUN: %dafny /compile:0 /autoTriggers:0 "%s" > "%t"
+// RUN: %dafny /compile:0 /deprecation:0 /autoTriggers:0 "%s" > "%t"
```

Comment:
Oh, I see now. It's because there are *so* many such files. Okay, I'm fine with those being changed in a future PR. But for now, I suggest reverting the addition ot `/deprecation:0` in those cases when adding that flag removes the previous warnings.

Reply:
Yes, there are quite a few files with quite a few semicolon errors. However there were only 5 files that had changes in the expected result by adding /deprecation:0 (because they suppressed other warnings). I did the work of removing semicolons from 4 of them and the /deprecation:0, and there now there is no change to expected result.

The 5th is `server/symbols.transcript`, which I do not have the tools to edit.
And there are ~100 files remaining with /deprecation:0